### DISCUSSION TITLE
How to autostart?

### DISCUSSION THREAD

**Discussion by sirdich** [UnknownSuperficialNight/nvidia-fan-control#4] [Q&A]
Can you please include in the docs the process of auto starting? You mention in the readme it's possible, but I haven't been able to figure our how to start a Rust executable on login via my own searching.

**Comment by UnknownSuperficialNight** [comment]
I made a quick [wiki](https://github.com/UnknownSuperficialNight/nvidia-fan-control/wiki/Auto%E2%80%90starting-the-Application-on-System-Login). Let me know if you get stuck or if you figure it out another way. (So i can add it to the wiki)

**Reply by sirdich** [reply]
Thanks, those where the 3 methods I've tried before. Crontab doesn't seem to work after testing, and systemd gives errors when starting. It's late for me, I can get you specifics later if you want them.

**Reply by UnknownSuperficialNight** [reply]
You'll likely need to mess around with it a bit to get everything functioning properly. The program will work with no issues as-long as you execute as root
